index.html

```diff
@@ -78,6 +78,11 @@ <h1>Moonlight</h1>
 					</li>
 					<li><a href="#">Host Downloads</a>
 						<ul>
+							<li>
+								<a href="https://github.com/LizardByte/Sunshine/releases">
```

Comment:
This should probably link to our docs or readme as not all packages are available from GitHub releases, such as Flatpak/Flathub, Fedora, homebrew, and pacman. And it's not really clear how to install or setup things by looking at the releases.

https://github.com/LizardByte/Sunshine/blob/master/README.md

OR

https://docs.lizardbyte.dev/projects/sunshine/en/latest/

... or even better yet to our Sunshine product page https://app.lizardbyte.dev/Sunshine, which has all relevant links (docs and downloads). (Flathub being added in https://github.com/LizardByte/Sunshine/pull/3366)